packages/hooks/src/useCFDDemoAccounts.ts

```diff
@@ -0,0 +1,18 @@
+import { useStore } from '@deriv/stores';
+
+const useCFDAccounts = () => {
```

Comment:
@shayan-deriv I think this file content is the same with `packages/hooks/src/useCFDAccounts.ts` which is wrong, Can you double check please 🙏🏻

Reply:
correct. updated the code